### PR TITLE
Add 500 new words to Spanish-Catalan bidix (unknown Catalan module words)

### PR DESCRIPTION
**Pull request by albertonl**
See https://github.com/apertium/apertium-spa-cat/pull/2 for more information.

### CODE REVIEW

File: apertium-cat.cat.dix [hunk 1]
```diff
@@ -120356,6 +120356,207 @@
...
+<e lm="regalimar" a="albertonl">   <i>regalimar</i><par n="abander/ar__vblex"/></e>
+<e lm="ruboritzar" a="albertonl">   <i>ruboritz</i><par n="abander/ar__vblex"/></e>
+<e lm="sollevar" a="albertonl">   <i>sollev</i><par n="abander/ar__vblex"/></e>
+<e lm="talonejar" a="albertonl">   <i>talone</i><par n="agreu/jar__vblex"/></e>
+<e lm="tintinar" a="albertonl">   <i>tintin</i><par n="abander/ar__vblex"/></e>
+<e lm="trampejar" a="albertonl">   <i>trampe</i><par n="agreu/jar__vblex"/></e>
+<e lm="xapurrejar" a="albertonl">   <i>xapurre</i><par n="agreu/jar__vblex"/></e>
+<e lm="xafarranxo" a="albertonl">   <i>xafarranxo</i><par n="abric__n"/></e>
+<e lm="vitualla" a="albertonl">   <i>vitualla</i><par n="abell/a__n"/></e>
+<e lm="violó" a="albertonl">   <i>violó</i><par n="acci/ó__n"/></e>
```
[the remark this answers is not in the document]
Comment (albertonl) [reply apertium-cat.cat.dix]: Changed. Thanks

File: apertium-cat.cat.dix [hunk 2]
```diff
@@ -120356,6 +120356,207 @@
...
+<e lm="proctòleg" a="albertonl">   <i>proctòl</i><par n="biòl/eg__n"/></e>
+<e lm="immunòleg" a="albertonl">   <i>immunòl</i><par n="biòl/eg__n"/></e>
+<e lm="grafòleg" a="albertonl">   <i>grafòl</i><par n="biòl/eg__n"/></e>
+<e lm="futuròleg" a="albertonl">   <i>futuròl</i><par n="biòl/eg__n"/></e>
+<e lm="estomatòleg" a="albertonl">   <i>estomatòl</i><par n="biòl/eg__n"/></e>
+<e lm="abduir" a="albertonl">   <i>abdu</i><par n="addu/ir__vblex"/></e>
+<e lm="arrecular" a="albertonl">   <i>arrecul</i><par n="abander/ar__vblex"/></e>
+<e lm="arrepapar" a="albertonl">   <i>arrepap</i><par n="abander/ar__vblex"/></e>
+<e lm="belluguet" a="albertonl">   <i>belluguet</i><par n="abric__n"/></e>
+<e lm="beudat" a="albertonl">   <i>beudat</i><par n="accessibilitat__n"/></e>
```
[the remark this answers is not in the document]
Comment (albertonl) [reply apertium-cat.cat.dix]: Changed
I'm going to upload the changes when i finish changing all in local

File: apertium-cat.cat.dix [hunk 3]
```diff
@@ -120356,6 +120356,207 @@
...
+<e lm="bibliòfil" a="albertonl">   <i>bibliòfil</i><par n="senyor__n"/></e>
+<e lm="bilionari" a="albertonl">   <i>bilionari</i><par n="senyor__n"/></e>
```
[the remark this answers is not in the document]
Comment (albertonl) [reply apertium-cat.cat.dix]: Done

File: apertium-cat.cat.dix [hunk 4]
```diff
@@ -120356,6 +120356,207 @@
...
+<e lm="llebeig" a="albertonl">   <i>llebeig</i><par n="abric__n"/></e>
+<e lm="lliurepensament" a="albertonl">   <i>lliurepensament</i><par n="abric__n"/></e>
+<e lm="mostatxo" a="albertonl">   <i>mostatxo</i><par n="abric__n"/></e>
+<e lm="noiet" a="albertonl">   <i>noiet</i><par n="senyor__n"/></e>
+<e lm="òbol" a="albertonl">   <i>òbol</i><par n="abric__n"/></e>
+<e lm="oblació" a="albertonl">   <i>oblaci</i><par n="acci/ó__n"/></e>
+<e lm="petitesa" a="albertonl">   <i>petitesa</i><par n="multimèdia__adj"/></e>
+<e lm="porticó" a="albertonl">   <i>portic</i><par n="acci/ó__n"/></e>
+<e lm="postilló" a="albertonl">   <i>postill</i><par n="acci/ó__n"/></e>
+<e lm="rutherfordi" a="albertonl">   <i>rutherfordi</i><par n="abanderric__n"/></e>
```
[the remark this answers is not in the document]
Comment (albertonl) [reply apertium-cat.cat.dix]: Apologize for the mistake

File: apertium-cat.cat.dix [hunk 5]
```diff
@@ -120356,6 +120356,207 @@
...
+<e lm="estomatòleg" a="albertonl">   <i>estomatòl</i><par n="biòl/eg__n"/></e>
+<e lm="abduir" a="albertonl">   <i>abdu</i><par n="addu/ir__vblex"/></e>
+<e lm="arrecular" a="albertonl">   <i>arrecul</i><par n="abander/ar__vblex"/></e>
+<e lm="arrepapar" a="albertonl">   <i>arrepap</i><par n="abander/ar__vblex"/></e>
+<e lm="belluguet" a="albertonl">   <i>belluguet</i><par n="abric__n"/></e>
+<e lm="beudat" a="albertonl">   <i>beudat</i><par n="accessibilitat__n"/></e>
+<e lm="bibliòfil" a="albertonl">   <i>bibliòfil</i><par n="senyor__n"/></e>
+<e lm="bilionari" a="albertonl">   <i>bilionari</i><par n="senyor__n"/></e>
+<e lm="biplaça" a="albertonl">   <i>biplaça</i><par n="multimèdia__adj"/></e>
+<e lm="cassolà" a="albertonl">   <i>cassol</i><par n="afric/à__adj"/></e>
```
[the remark this answers is not in the document]
Comment (albertonl) [reply apertium-cat.cat.dix]: Changed all of them. Thank you for your comments and review

File: apertium-cat.cat.dix [hunk 6]
```diff
@@ -120356,6 +120356,220 @@
...
+<e lm="biplaça" a="albertonl">   <i>biplaça</i><par n="multimèdia__adj"/></e>
+<e lm="turbopropulsió" a="albertonl">   <i>turbopropulsi</i><par n="acci/ó__n"/></e>
+<e lm="telebotiga" a="albertonl">   <i>telebotig</i><par n="alg/a__n"/></e>
+<e lm="sotsintendent" a="albertonl">   <i>sotsintendent</i><par n="abric__n"/></e>
+<e lm="solideu" a="albertonl">   <i>solideu</i><par n="abric__n"/></e>
+<e lm="sagal" a="albertonl">   <i>sagal</i><par n="abric__n"/></e>
+<e lm="saurí" a="albertonl">   <i>saurí</i><par n="abric__n"/></e>
+<e lm="semidéu" a="albertonl">   <i>semidéu</i><par n="abric__n"/></e>
+<e lm="cuca" a="albertonl">   <i>cu</i><par n="acústi/ca__n"/></e>
+<e lm="culgròs" a="albertonl">   <i>culgr</i><par n="aliment/ós__adj"/></e>
```
Comment (jaumeortola) [inline apertium-cat.cat.dix]: paradigm: capgr/òs__adj

File: apertium-cat.cat.dix [hunk 7]
```diff
@@ -120356,6 +120356,220 @@
...
+<e lm="enllustrador" a="albertonl">   <i>enllustrador</i><par n="senyor__n"/></e>
+<e lm="escrivà" a="albertonl">   <i>escriv</i><par n="bacall/à__n"/></e>
+<e lm="garbelladures" a="albertonl">  <i>garbelladures</i><par n="escombraries__n"/></e>
+<e lm="gemmòleg" a="albertonl">   <i>gemmòl</i><par n="biòl/eg__n"/></e>
+<e lm="herbolari" a="albertonl">   <i>herbolari</i><par n="senyor__n"/></e>
+<e lm="io-io" a="albertonl">   <i>io-io</i><par n="abric__n"/></e>
+<e lm="licàntrop" a="albertonl">   <i>licàntrop</i><par n="senyor__n"/></e>
+<e lm="llebeig" a="albertonl">   <i>llebeig</i><par n="abric__n"/></e>
```
Comment (jaumeortola) [inline apertium-cat.cat.dix]: paradigm: rebu/ig__n

File: apertium-cat.cat.dix [hunk 8]
```diff
@@ -120356,6 +120356,220 @@
...
+<e lm="beutat" a="albertonl">   <i>beutat</i><par n="accessibilitat__n"/></e>
+<e lm="bibliòfil" a="albertonl">   <i>bibliòfil</i><par n="senyor__n"/></e>
+<e lm="bilionari" a="albertonl">   <i>bilion</i><par n="adjudicat/ari__n"/></e>
+<e lm="biplaça" a="albertonl">   <i>biplaça</i><par n="multimèdia__adj"/></e>
+<e lm="turbopropulsió" a="albertonl">   <i>turbopropulsi</i><par n="acci/ó__n"/></e>
+<e lm="telebotiga" a="albertonl">   <i>telebotig</i><par n="alg/a__n"/></e>
+<e lm="sotsintendent" a="albertonl">   <i>sotsintendent</i><par n="abric__n"/></e>
+<e lm="solideu" a="albertonl">   <i>solideu</i><par n="abric__n"/></e>
+<e lm="sagal" a="albertonl">   <i>sagal</i><par n="abric__n"/></e>
+<e lm="saurí" a="albertonl">   <i>saurí</i><par n="abric__n"/></e>
```
Comment (jaumeortola) [inline apertium-cat.cat.dix]: boc/í__n

**Pull request merged**